docs/docs/manuals/error-messages.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Error Messages
+---
+
+You will find some possible error messages here, what the problem is and what some possible solutions can be.
+
+## User Agent does not correspond
+
+This error appeared for some users as soon as they were redirected to the login page of ZITADEL.
+
+We only found this issue with iPhone users and it was dependent on the settings of the device.
+
+### Solution
```

Comment:
I would recommend that we also provide a problem statement. E.g cookies are blocked which affects ZITADELs function

Reply:
i think this makes sense, I also added why we need the cookies